src/DocTests.jl

```diff
@@ -401,7 +401,8 @@ function report(result::Result, str, doc::Documents.Document)
 
         $(result.output)
 
-        """, diff, _file=result.file, _line=lines[1])
+        """, diff; _file=result.file,
+          _line=lines === nothing ? (@__LINE__) : first(lines))
```

Comment:
This should also work, both this and your suggestion are equally useless I guess.
```suggestion
          _line=lines === nothing ? nothing : first(lines))
```

Reply:
I thought maybe downstream loggers will expect `_line` will only be passed when it has an actual number, and I couldn't figure how to nicely conditionally pass it (could just write the whole macro twice but that seems less ideal), so I thought to pass a valid but useless line number instead of an invalid one.

In particular, the GitHubActionsLogger will do
```julia
julia> @warn "Hi" _line=nothing
::warning file=REPL[7],line=::Hi

julia> @warn "Hi"
::warning file=REPL[8],line=1::Hi
```
and the first one doesn't really look valid, though I suppose GitHubActions itself won't throw on seeing it lol. I can fix that in GitHubActionsLogger but it made me think that in general loggers might not expect `_line=nothing`.

Reply:
I have no idea why CI is failing so I’ll try with `nothing`